upcase_booklist_app/templates/author.html

```diff
@@ -0,0 +1,45 @@
+<!doctype html>
+<html>
+
+    <head>
+        <title>Curious Programmer | Author</title>
+        <link rel="stylesheet" href="{{ url_for('static', filename='main.css') }}">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    </head>
+
+    <body> 
+        <nav class="page-header">
+            <div class="header-items"><a href="{{ url_for('index') }}">Home</a></div> 
+            <div class="header-items"><a href="{{ url_for('book_list') }}">Books</a></div>
+            <div class="header-items"><a href="{{ url_for('authors') }}">Authors</a></div>
+            <div class="header-items"><a href="{{ url_for('lists') }}">Lists</a></div>
+        </nav>
+    <body> 
+
+    
+        <div>
+            <div class="container">
+                
+                <div class="author-name">
+                    <h1>{{ author_object['name'] }}</h1>
+                    <p class="text">{{ author_object['bio'] }}</p>
+                </div>
+
+                <div class="author-book">
+                    {% for book in author_object['books'] %}
+
+                        <div class="book-image">
+                            <a href="{{ url_for('book', book_id=book['book_id']) }}">
```

Comment:
good use of having two variables for this template: an author and a book. And that's all you need!